### ISSUE TITLE
Version mismatch library

### ISSUE THREAD

**Issue by libremente** [italia/developers-italia-onboarding#29]
In console this warning appears:
```
Version mismatch when loading serialised index. Current version of lunr '2.2.1' does not match serialized index '2.3.5'
```

**Comment by libremente** [comment]
Probably upgrading the `lunr` version may fix.
ncu suggests:
```
lunr                        ^2.2.1  →   ^2.3.5
```

**Comment by giuseppe-santoro** [comment]
The new `lunr` version is not needed to run this project so you could simply fix the version `2.2.1`  https://github.com/olivernn/lunr.js/blob/master/CHANGELOG.md

If you want to use the new `lunr` version, you should delete the files `authorities.index.json` and `authorities.db.json` and run the server. It should create the new index https://github.com/italia/onboarding.developers.italia.it/blob/master/server.js#L89

**Comment by libremente** [comment]
Thanks @giuseppe-santoro for the reply! I will check and let you know asap :+1:

**Comment by libremente** [comment]
Fixed in #37. Closing.
